pkg/utils/proxy/cloudproxy.go

```diff
@@ -104,14 +106,24 @@ func (cp *CloudProxy) deploymentSpec() (v1apps.DeploymentSpec, error) {
 
 	terminationGracePeriodSeconds := int64(120) // force kill pod after this time
 
+	var annotations map[string]string
+
+	if cp.MonitoringEnabled {
+		annotations = map[string]string{
+			"prometheus.io/scrape": "true",
```

Comment:
(minor)

prometheus is default to scape path "/metrics". But it's possible to overwrite this with this annotation `prometheus.io/path`.

but think, if somebody need this, it's a simple addition.

Reply:
Agreed, in the long run we could add a loosely typed proxyAnnotations map. But YAGNI anytime soon.